homework/style.css

```diff
@@ -1,3 +1,114 @@
+* {
```

Comment:
You probably will not win first prize for your UI design (nor will I, for that matter). On a mobile, your select box seems to run off the right margin:

![image](https://user-images.githubusercontent.com/2788771/62132808-08b3dd80-b2de-11e9-8e4f-7d99ec25bc9e.png)